[19.0][MIG] hr_timesheet_begin_end: Migration to 19.0

bizzappdev opened the pull request.

CRogos committed:
[MIG] hr_timesheet_activity_begin_end: Update to 15.0

Commit:
[IMP] hr_timesheet_begin_end: black, isort, prettier

Commit:
[MIG] hr_timesheet_begin_end: Migration to 16.0

Commit:
[IMP] hr_timesheet_begin_end: missing form fields

[UPD] Update hr_timesheet_begin_end.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_begin_end
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_begin_end/

[UPD] README.rst

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (6 of 6 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_begin_end
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_begin_end/pt_BR/

pere-aquarian committed:
Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 100.0% (6 of 6 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_begin_end
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_begin_end/ca/

NICO-SOLUTIONS committed:
[IMP] hr_timesheet_begin_end: pre-commit auto fixes

NICO-SOLUTIONS committed:
[MIG] hr_timesheet_begin_end: Migration to 17.0

Commit:
[UPD] Update hr_timesheet_begin_end.pot

BhaveshHeliconia committed:
[IMP] hr_timesheet_begin_end: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] hr_timesheet_begin_end: Migration to 18.0

Commit:
[UPD] Update hr_timesheet_begin_end.pot

OCA-git-bot committed:
[BOT] post-merge updates

CRogos committed:
[UPD] hr_timesheet_begin_end: add a hint for project_timesheet_time_control

OCA-git-bot committed:
[BOT] post-merge updates

remytms commented:
Did you take a look at [this fix](https://github.com/OCA/timesheet/pull/816) in 18.0 ?

CRogos commented:
I would suggest to merge this with this https://github.com/OCA/timesheet/pull/833 module.

NICO-SOLUTIONS commented:
Just wanted to politely ask about the current status and whether this is still being actively maintained or planned to be continued, considering the mentioned fix.

Happy to help or provide additional context if needed.

bizzappdev commented:
@CRogos, Not sure I am missing anything, but I doubt that the PR for the fix you are mentioning is unrelated to this module/PR.
Our PR is for  `hr_timesheet_begin_end`, and the fix PR is for `hr_timesheet_time_control_begin_end`

@NICO-SOLUTIONS We will be doing the fix for another PR soon.

CRogos commented:
@bizzappdev this is correct. There are currently two worlds in OCA. hr_timesheet_time_control (former project_time_control) + hr_timesheet_time_control_begin_end addresses the same goal as hr_timesheet_begin_end, but has a much better integration. 
I've recently moved from hr_timesheet_begin_end to hr_timesheet_time_control_begin_end and because I wasn't aware of project_time_control which is a much cooler modules, I do some promoting ;)

bertstomphorst committed:
Validate 'overlap' on employee instead of user

The validation for overlapping times was done on user_id, which is the user entering the lines. This should be on employee_id, the employee *for who* the times are entered.

OCA-git-bot committed:
[BOT] post-merge updates

bizzappdev committed:
[MIG] hr_timesheet_begin_end: Migration to 19.0

alexey-pelykh reviewed (approved):
Looks solid. Straightforward 19.0 migration — overlap validation logic and time fields are clean. LGTM.

AEstLo reviewed (approved):
LGTM - code review

WesleyOliveira98 reviewed (approved):
LGTM

pedrobaeza commented:
/ocabot migration hr_timesheet_begin_end
/ocabot merge nobump

pedrobaeza commented:
/ocabot merge nobump
